src/components/ItemList/index.js

```diff
@@ -59,64 +62,165 @@ export function ItemList({ items }) {
+      body: item._id,
+    })
+    result
+      .then(response => response.json())
+      .then(responseData => {
+        alert('Delete Success!!')
+        window.location.reload() 
+      })
+      .catch(function(err) {
+        console.log(err)
+        alert(err)
+      })
+  }
+  function CustomerButtons({index}) {
+    return (
+      <>
+        <CardActions>
+        <Button
+          size="small"
+          color="primary"
+          aria-controls="simple-menu"
+          aria-haspopup="true"
+          key={index}
+          onClick={handleClickPrice.bind(this, index)}
+        >
+          Buy
+        </Button>
+        <Modal
+          aria-labelledby="simple-modal-title"
+          aria-describedby="simple-modal-description"
+          open={openPrice}
+          onClose={handleClickAwayPrice}
+        >
+          <div className={classes.paper}>
+            <h1>Excelent choice!</h1>
+            <h2>Owner Name </h2> <p>{itemIndex.ownerName}</p>
+            <h2>Owner Mail </h2>
+            <p>{itemIndex.ownerMail}</p>
+          </div>
+        </Modal>
+      </CardActions>
+      </>
+    )
+  }
+  function DeleteButtons({item}) {
+    return (
+      <>
+        <CardActions>
+        <Button
+          size="small"
+          color="primary"
+          aria-controls="delete-menu"
+          aria-haspopup="true"
+          key={item}
+          onClick={handleClickDelete.bind(this, item)}
+        >
+          Delete
+        </Button>
+        <Modal
+          aria-labelledby="delete-modal-title"
+          aria-describedby="delete-modal-description"
+          open={openDelete}
+          onClose={handleClickAwayDelete}
+        >
+          <div className={classes.paper}>
+            <h1>Delete</h1>
+            <h2>Owner Name </h2> <p>{item._id}</p>
+          </div>
+        </Modal>
+      </CardActions>
+      </>
+    )
+  }
+  function AuthedButtons({auth,item,index}) {
+    return (
+      <>
+      {auth.Uid==item.ownerId ? <DeleteButtons item={item}/> : <CustomerButtons  index={index}/>}
+      </>
+    )
+  }
+  function UnAuthedButtons() {
+    return (
+      <>
+      <CardActions>
+      <Button
+        size="small"
+        color="primary"
+        aria-controls="simple-menu"
+        aria-haspopup="true"
+        onClick={unauthhandleClickPrice.bind()}
+      >
+        Buy
+      </Button>
+      <Modal
+        aria-labelledby="simple-modal-title"
+        aria-describedby="simple-modal-description"
+        open={openPrice}
+        onClose={handleClickAwayPrice}
+      >
+        <div className={classes.paper}>
+          <h1>Please Log In First</h1>
+        </div>
+      </Modal>
+    </CardActions>
+      </>
+    )
+  }
   const handleClickAway = () => setOpenCard(false)
   const handleClickAwayPrice = () => setOpenPrice(false)
+  const handleClickAwayDelete = () => setOpenDelete(false)
+  const { auth } = useUserContext()
 
   return (
-    <div className={classes.root}>
+    <>
```

Comment:
I am wondering why we change this line to `<>`?
Are we changing the style of items?